examples/incremental_learning/helmet-detection-modelbox/incremental_learning-modelbox/src/flowunit/image_shape/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
src/flowunit/helm_infer/saved_model.pb  is large file, and it should not be committed to this repository.
It is suggestted to put the file to GoogleDrive or elsewhere, and put the link here.